scripts/plugins/storage/postgres/init.sql

```diff
@@ -602,6 +602,8 @@ CREATE TABLE fledge.users (
        enabled           boolean                     NOT NULL DEFAULT TRUE,
        pwd_last_changed  timestamp(6) with time zone NOT NULL DEFAULT now(),
        access_method     character varying(5) CHECK( access_method IN ('any','pwd','cert') )  NOT NULL DEFAULT 'any',
+       failed_attempts   integer                     DEFAULT 0,
+       block_until       timestamp(6)                DEFAULT now(),
```

Comment:
with timezone?

Reply:
Probably best tp use etc as the user might change timezones, especially should not be based on the timezone of the caller of the API.

Reply:
@MarkRiddoch While login user can only provide id and password. There is no way user can provide any information related to time. API does all the calculation for block time on the basis of information stored into the database. So we are safe here even if user changes timezone. so is it good now?